code/game/machinery/autolathe.dm

```diff
@@ -233,32 +233,31 @@
 	if(href_list["make"] && machine_recipes)
 		var/index = text2num(href_list["make"])
 		var/multiplier = text2num(href_list["multiplier"])
-		var/datum/autolathe/recipe/making
+		build_item = null
 
 		if(index > 0 && index <= machine_recipes.len)
```

Comment:
```suggestion
		if(index in 0 to machine_recipes.len)
```

Imo this is more readable. Can change in other places like line 242 too if you prefer this.